Readme.md

```diff
@@ -240,55 +183,55 @@ final class SomethingHappenedElsewhere implements EventInterface, ProvidesEventT
 }
 ```
 
-### Generic Fault (WIP)
-
-* [x] **GenericFault**: event triggered by the EventBus is an event handler throw an exception
-
-### How to implement you own event listener ?
-
-Your must implement the ```EventListenerInterface```:
+### CommandHandlers
 
 ```php
-class ConsoleOutputListener implements EventListenerInterface
+/**
+ * @Flow\Scope("singleton")
+ */
+final class ButtonCommandHandler
 {
     /**
-     * @var SystemLoggerInterface
+     * @var ButtonRepository
      * @Flow\Inject
      */
-    protected $systemLogger;
+    protected $buttons;
 
     /**
-     * @param ButtonTagged $event
-     * @param EventMetadata $metadata
+     * @param CreateButton $command
      */
-    public function onButtonTagged(ButtonTagged $event, EventMetadata $metadata)
+    public function handleCreateButton(CreateButton $command)
     {
-        $this->systemLogger->log('--- ConsoleOutputListener say something has been tagged ---');
+        $button = new Button($command->getIdentifier());
+        $button->changeLabel($command->getLabel());
+
+        $this->buttons->save($button);
     }
 }
 ```
 
-The event handler locator can throw an exception if something is wrong with your command handler definition. In that
-case please check your system log for more information.
+The CommandHandler is currently not a formal part of the Framework, but is a recommended abstraction to introduce
+in order to easily support dispatching commands through other entry points (e.g. CLI). It might be introduced (again)
+later on in combination with an CommandBus to provide a single (asynchronous) dispatch entry point.
 
-All the wiring between event is done automatically, if you respect the following convention:
+### Aggregates
 
-* Method name must be on[ShortEventName]
-* The first parameter must be casted with your ```Event```
-* The second parameter is optional, but should be casted to ```EventMetadata```
+t.b.w.
 
-## EventStore
+### ProcessManagers
 
-See package **Neos.EventSourcing**.
+t.b.w.
 
-## Message
+### EventPublisher
 
-Messaging infrastructure, base class/traits to build your own Events.
+t.b.w.
 
-## Projections
+# Query / Read side
 
-Because a projection is usually tailored to one or more views in an application, we recommend using the
-`Application\Projection\[ProjectionName]` namespace for the respective code.
+Your read side is mainly made up of one or multiple Projections and all the Application logic around them,
+e.g. the typical Flow MVC components like Controller(s), DTOs, Templates and Layouts.
```

Comment:
The application logic is best placed along the read side. You need an entry point to your queries (= Controller) and you most likely want to return some HTML there while your write side should not return any HTML at all. But I'm totally open to rephrasing this.